applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -20,21 +20,23 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_LED_LOG_LEVEL);
 
-
 struct led {
 	struct device *pwm_dev;
 
 	size_t id;
 	struct led_color color;
 	const struct led_effect *effect;
+	const struct led_effect *next_effect;
+	bool stream;
 	u16_t effect_step;
 	u16_t effect_substep;
 
 	struct k_delayed_work work;
 };
 
+static const struct led_effect *leds_state_bck[CONFIG_DESKTOP_LED_COUNT];
 static struct led leds[CONFIG_DESKTOP_LED_COUNT];
-
+static bool streaming;
```

Comment:
Use one static bool, or bool per LED, no need both

Reply:
for now I left one static bool